docs/configuring-socorro.rst

```diff
@@ -1,25 +1,68 @@
 .. index:: configuring-socorro
 
-Configuring Socorro
-===================
+Configure and run Socorro
+=========================
 
-Socorro produces graphs and reports, most are updated once per day.
+Initial setup
+-------------
 
-You must enter information about your releases into Socorro in order
-for this to work, and this information must match the incoming crashes.
+Socorro provides a setup script which attempts to initialize everything.
+If you have a distributed setup you should review this script and run
+the appropriate commands on the right servers::
```

Comment:
This will need to be amplified; for example, what does "the right servers" mean, and how does one know which commands belong to which servers?  The former question could be answered by a (a link to the) section on the different roles, while the latter would be addressed by a ordering and verbosely commenting the setup script.